src/CMakeLists.txt

```diff
@@ -107,6 +107,8 @@ add_executable(
   triton_signal.h
 )
 
+set(CMAKE_CXX_STANDARD_REQUIRED True)
```

Comment:
This value must be part of top level instructions.

Reply:
Moved instructions accordingly.